### ISSUE TITLE
Cannot tab to Close button

### ISSUE THREAD

**Issue by bgurney** [getsentry/sentry#15919]
There is an accessibility issue for keyboard access in that you cannot tab to the Close button (or use Esc key to close it). This means that the dialog is stuck open and users cannot use site navigation to go anywhere else.

**Comment by cameronmcefee** [comment]
Thanks @bgurney. I've transferred this issue to the correct repo and let the team know.

**Comment by BYK** [comment]
Sorry it took so long for us to get to. We'll be looking into this soon as I just verified this is still the case.

**Comment by BYK** [comment]
Fixed by #24151. Thanks @markstory!
